src/examples/px4_simple_app/px4_simple_app.c

```diff
@@ -55,75 +55,75 @@ __EXPORT int px4_simple_app_main(int argc, char *argv[]);
 
 int px4_simple_app_main(int argc, char *argv[])
 {
-	PX4_INFO("Hello Sky!");
```

Comment:
Why did you convert to spaces, as we use tabs in the repo? - Which is also why CI fails

Reply:
We really use tabs? OK, sorry, just a mistake. Will revert, or create new PR without this.